spec.html

```diff
@@ -41909,7 +41918,7 @@ <h1>Atomics.wait ( _typedArray_, _index_, _value_, _timeout_ )</h1>
         1. Let _block_ be _buffer_.[[ArrayBufferData]].
         1. Let _WL_ be GetWaiterList(_block_, _indexedPosition_).
         1. Perform EnterCriticalSection(_WL_).
-        1. Let _elementType_ be the Element Type value in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for _arrayTypeName_.
+        1. Let _elementType_ be TypedArrayElementType(_typedArray_).
```

Comment:
For cases like this, `_arrayTypeName_` is now only used once, and we can just inline it. Same above on line 41482.

Reply:
Done, in ValidateIntegerTypedArray, Atomics.store, and Atomics.wait.